tests/migrations/test_autodetector.py

```diff
@@ -4346,6 +4346,39 @@ def test_fk_dependency_other_app(self):
             changes, "testapp", 0, [("otherapp", "__first__")]
         )
 
+    def test_alter_unique_together_fk_to_m2m(self):
+        changes = self.get_changes(
+            [self.author_name, self.book_foo_together],
```

Comment:
Can we add `book_unique_together` without `index_together` (that we want to deprecate)?

Reply:
Done ✔️